pyproject.toml

```diff
@@ -391,6 +391,7 @@ lint.ignore = [  # NOTE: non-permanent exclusions should be added to `.ruff.toml
 [tool.ruff.lint.extend-per-file-ignores]
 "setup.py" = ["INP001"]  # Part of configuration, not a package.
 ".github/workflows/*.py" = ["INP001"]
+"astropy/cosmology/**/*.py" = ["RUF022"]
```

Comment:
Apologies, I had this marked as viewed. I think we discussed just ignoring the files with manually sorted `__all__`, not the whole module.

Reply:
No worries. Am I right to assume that "`__all__`s with comments from gh-14953" is the exact same set ? I'll push with this interpretation, but please do correct me if needed !

Reply:
I think just the top and flrw inits

Reply:
Thanks!